### ISSUE TITLE
Support for CogVideoX-2b

### ISSUE THREAD

**Issue by bil-ash** [leejet/stable-diffusion.cpp#379]
Please add support for CogVideoX-2b[](https://huggingface.co/THUDM/CogVideoX-2b) since its license is now open source and also it has quite good quality in a small size

**Comment by Amin456789** [comment]
@leejet yes please work on this  leejet, cogvideox is like flux in prompt following and quality is great, we need this fo be quantized for low ram and cpu users
i can see u made everything important for sd cpp, i ts great time for a txt to video i guess, it will be great to have  this one instead svd and the others, please consider working on this one, thank u
